cli/http_body.rs

```diff
@@ -87,26 +87,3 @@ impl AsyncRead for HttpBody {
     }
   }
 }
-
-#[test]
-fn test_body_async_read() {
```

Comment:
Unfortunately all methods of `Decoder` are private or crate scoped, so there's no way to construct it manually...